src/interfaces/IAdaptiveCurveIrm.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.5.0;
+
+import {IIrm} from "../../lib/morpho-blue/src/interfaces/IIrm.sol";
+import {Id} from "../../lib/morpho-blue/src/interfaces/IMorpho.sol";
+
+interface IAdaptiveCurveIrm is IIrm {
```

Comment:
Be consistent with the natspec you already have and add 

```
/// @title IAdaptiveCurveIrm
/// @author Morpho Labs
/// @custom:contact security@morpho.org
```

Reply:
Fixed in #113